README.md

```diff
@@ -1 +1,37 @@
-# EXPRESS-STARTER
+# CLUEWORDS
+A web app that allows users to play a game of "Codenames" together online. 
+
+[Codenames](https://en.wikipedia.org/wiki/Codenames_(board_game)) is "a 2015 card game for 4–8 players... (where) two teams compete by each having a "spymaster" give one-word clues that can point to multiple words on the board. The other players on the team attempt to guess their team's words while avoiding the words of the other team".
+
+**Live demo:** https://cluewords.herokuapp.com/
+
+**Features**
+* Sign up/Login authentication
+* Private game lobbies
+* A team voting system for player guesses
+* Real-time chat between players of the same team
+* Game state persistence during a game
+
+## Contributors
+* [Tony Chen](https://github.com/tonyc856)
+* [Renyi Cao](https://github.com/h0vscat)
```

Comment:
Sorry Tony, Could you please update my link. I just update my github username to make my online identity more consistent. lol